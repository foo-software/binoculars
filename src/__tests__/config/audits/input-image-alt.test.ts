import auditInputImageAlt from '../../../config/audits/input-image-alt';

describe('input-image-alt audit', () => {
  it('should match snapshot', () => {
    expect(auditInputImageAlt('en')).toMatchSnapshot();
  });
});
