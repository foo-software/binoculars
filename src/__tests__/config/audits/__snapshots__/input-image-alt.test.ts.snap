// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input-image-alt audit should match snapshot 1`] = `
Object {
  "description": "When an image is being used as an \`<input>\` button, providing alternative text can help screen reader users understand the purpose of the button. [Learn more](https://web.dev/input-image-alt/).",
  "title": "\`<input type=\\"image\\">\` elements have \`[alt]\` text",
}
`;
